artiq/coredevice/phaser.py

```diff
@@ -468,6 +474,32 @@ def get_crc_err(self):
         """
         return self.read8(PHASER_ADDR_CRC_ERR)
 
+    @kernel
+    def measure_frame_timestamp(self):
+        """Perform a register read and record the exact frame timing in `self.frame_tstamp`.
+
+        Deterministic timing requires updates to be schedule at multiples of `self.t_frame` later.
```

Comment:
Only true for for updates that go through the register interface.

Reply:
Changed in 075cb26